app/controllers/instrument_issues_controller.rb

```diff
@@ -37,9 +43,29 @@ def init_product
   end
 
   def init_instrument_issue
-    @instrument_issue = InstrumentIssue.new(product: @product, 
+    @instrument_issue = InstrumentIssue.new(product: @product,
                                             user: current_user,
                                             order_detail: @order_detail)
   end
 
+  def redirect_to_path
+    redirect_to_order_id = params[:redirect_to_order_id]
+
+    if redirect_to_order_id.present?
```

Comment:
should we be checking `modal?` here?  
can you add a comment: `# this means the request came from a modal on the order show page`

Reply:
The request for `create` returns `false` for `request.xhr?`, so I had to remove the `modal?` condition from here.